diff --git a/fusion/src/init.lua b/fusion/src/init.lua
index 79e91d0..9cd3f51 100644
--- a/fusion/src/init.lua
+++ b/fusion/src/init.lua
@@ -46,7 +46,7 @@ type Fusion = {
 }
 
 return restrictRead("Fusion", {
-	version = {major = 0, minor = 2, isRelease = true},
+	version = {major = 1234, minor = 2, isRelease = true}, --did some change to major
 
 	New = require(script.Instances.New),
 	Hydrate = require(script.Instances.Hydrate),
